src/commands/console/workspace/download.js

```diff
@@ -49,7 +49,7 @@ class DownloadCommand extends ConsoleCommand {
 
       let fileName = 'console.json'
       if (!flags.workspaceId && !flags.projectId && !flags.orgId) {
-        const projectName = this.getConfig(`${CONFIG_KEYS.PROJECT}.name`)
+        const projectName = this.getConfig(`${CONFIG_KEYS.PROJECT}.title`)
```

Comment:
nit: projectTitle?

Reply:
Actually, this one might need to still be project.name since it is being used as a file name
`${orgId}-${projectName}-${workspaceName}.json`
Titles can contain spaces and someday maybe even some other char, like ** My Awesome App ** which we would need to guard against in file names